test/nyc-integration.js

```diff
@@ -14,6 +14,8 @@ const rimraf = require('rimraf')
 const makeDir = require('make-dir')
 const spawn = require('child_process').spawn
 const si = require('strip-indent')
+// BUGBUG: when upgrading execa to 2.x stripEof is renamed to stripFinalNewline.
```

Comment:
should we not upgrade because of this? what's wrong?

Reply:
2.x isn't released yet and will require node.js 8.

Reply:
@bcoe has requested that we not use execa, for now I'm just going to use spawnSync.

Reply:
:+1: love to use Node's built-ins where possible, and it felt a bit weird that we were adding a caveat:

```
// BUGBUG: 
```

unrelated to our test; thanks for entertaining me being curmudgeonly.